src/login/components/LoginFormWalletFile/LoginFormWalletFile.js

```diff
@@ -47,22 +52,59 @@ export default class LoginFormWalletFile extends React.Component {
   }
 
   renderAccounts = () => {
-    const { accounts, wif } = this.props;
+    const { accounts, encryptedWIF } = this.props;
 
     if (accounts.length === 0) {
       return null;
     }
 
     return (
-      <Select className={styles.accounts} value={wif} onChange={this.handleSelect}>
-        <option value="">Select an account</option>
-        {map(this.props.accounts, (account, index) => (
-          <option value={account.encrypted} key={`account${index}`}>{account.label}</option>
-        ))}
-      </Select>
```

Comment:
I think we should have this function just return the `<Select>` as it was before, and call `this.renderPassphraseInput()` and `this.renderDescription()` directly from `render`.